recipes/sagemaker-pytorch-serving-container/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "sagemaker-pytorch-serving-container" %}
+{% set version = "1.0.5" %}
+{% set commit = "1ba29db3c90e3a71a708319693372ec2f51ef4fe" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/aws/{{ name }}/archive/{{ commit }}.tar.gz"
```

Comment:
Can’t you use the version tag instead of the commit? The updated bot won’t work this way

Reply:
The package currently doesn't have a release on github but I've created an issue on the page to generate one. In the meantime, I would like to get this in conda-forge as soon as possible and will update with the version tag as soon as the release is generated.

Reply:
IDK what our policy is on special cases like this. @conda-forge/core can we push versions as commits if upstream has not yet made a release?

Reply:
[https://github.com/conda-forge/nvidia-apex-feedstock/blob/master/recipe/meta.yaml](url) and [https://github.com/conda-forge/nltk_data-feedstock/blob/master/recipe/meta.yaml](url) both use versions as commits

Reply:
Alright. I would encourage a date-based version until something gets tagged.

Reply:
If we go for a date-based version, and there was a release 1.0.0, we can't switch. (unless we use  something like `1!1.0.0`)

Reply:
I would go for something like `0.0.1.dev0`

Reply:
Good point @isuruf! Have made an official policy on this? We might want to.

Reply:
+1 on the `0.0.1.dev0` or rather `0.0.1.dev0+1ba29db3` to give the user an indication where that non-official version is derived from.

IMHO, without any official release we should either
1. wait with merging this PR, or
2. at least put the package off of `conda-forge/label/main`. (cf. https://github.com/conda-forge/conda-forge-enhancement-proposals/pull/3)